samtranslator/feature_toggle/feature_toggle.py

```diff
@@ -57,11 +67,23 @@ def is_enabled_for_account_in_region(self, feature_name, stage, account_id, regi
         if not stage_config:
             LOG.info("Stage '{}' not enabled for Feature '{}'.".format(stage, feature_name))
             return False
-        account_config = stage_config.get(account_id) if account_id in stage_config else stage_config.get("default", {})
-        region_config = (
-            account_config.get(region, {}) if region in account_config else account_config.get("default", {})
-        )
-        is_enabled = region_config.get("enabled", False)
+
+        if account_id in stage_config:
+            account_config = stage_config.get(account_id)
+            region_config = (
+                account_config.get(region, {}) if region in account_config else account_config.get("default", {})
+            )
+        else:
+            region_config = stage_config.get(region, {}) if region in stage_config else stage_config.get("default", {})
+
+        if "enabled-%" in region_config:
+            # Percentage-based enablement
+            bound = region_config.get("enabled-%")
```

Comment:
Probably add few lines of comments explaining what bound and partition mean here.